definitions/divorce/json/CaseField/CaseField-general-referral-nonprod.json

```diff
@@ -84,5 +84,32 @@
     "Label": "General referral decision date",
     "FieldType": "Date",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "20/10/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "GeneralReferralPaymentDetails",
+    "Label": "How will payment be made?",
+    "FieldType": "FixedList",
+    "FieldTypeParameter": "PaymentTypeEnum",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "20/10/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "generalReferralPaymentFeeSummary",
```

Comment:
shouldn't you populate this field in callback?

Reply:
Ive followed the structure as it was made before. Am i missing something that `generalApplicationWithoutNoticeFeeSummary` had?

Reply:
Ho are you going to use this field?

Reply:
My intention was to use it the same way other similar ones have done. What would you recommend?